expirationd.lua

```diff
@@ -650,6 +656,28 @@ local function expirationd_run_task(name, space_id, is_tuple_expired, options)
         if expire_index.type ~= "TREE" and expire_index.type ~= "HASH" then
             error("Not supported index type, expected TREE or HASH")
         end
+        local engine = box.space[space_id].engine
+        if engine == "memtx" and expire_index.func ~= nil then
+            local supported = false
+            local sdk = rawget(_G, "_TARANTOOL")
```

Comment:
Nit: `sdk` is not appropriate term here IMHO. Just `version` would look better. Or `_G._TARANTOOL:split(<...>)`.